### ISSUE TITLE
migrations:continue should not create migrations table

### ISSUE THREAD

**Issue by hrach** [nextras/migrations#32]
Currently, if database does not have migrations table and you run migrations:continue, it creates migrations table and runs all your migrations. if your initial migrations isn't created correctly and you are in the middle of project, you can lost your data.

example of initial migrations (which may be an default output of some tools, i.e. Navicat):
```sql
DROP TABLE IF EXISTS `acl_acl`;
CREATE TABLE `acl_acl` (
...
)
```

**Comment by JanTvrdik** [comment]
It's intentional – migrations:continue should work the first time as migrations:reset (it simplifies deploy for example). We may consider a requirement that the database must be initially empty.

**Comment by hrach** [comment]
>  We may consider a requirement that the database must be initially empty.

Yeah, but that may be quite difficult. Though, I prefer the higher security over the usability.

**Comment by JanTvrdik** [comment]
If we ignore triggers, views, procedures and other complicated stuff and check only the count of tables, it should be fine.
